ibm/provider.go

```diff
@@ -356,6 +357,7 @@ func Provider() terraform.ResourceProvider {
 			"ibm_container_vpc_alb":                              resourceIBMContainerVpcALB(),
 			"ibm_container_vpc_worker_pool":                      resourceIBMContainerVpcWorkerPool(),
 			"ibm_container_vpc_cluster":                          resourceIBMContainerVpcCluster(),
+			"ibm_container_vpc_worker_volume_attachment":         resourceIBMContainerVpcWorkerVolumeAttachment(),
```

Comment:
can we rename ibm_container_vpc_worker_volume_attachment to ibm_container_vpc_worker_storage